staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2309,16 +2309,7 @@ func (d *CronJobDescriber) Describe(namespace, name string, describerSettings De
 		}
 		return describeCronJob(cronJob, events)
 	}
-
-	// TODO: drop this condition when beta disappears in 1.25
-	cronJobBeta, err := d.client.BatchV1beta1().CronJobs(namespace).Get(context.TODO(), name, metav1.GetOptions{})
-	if err != nil {
-		return "", err
-	}
-	if describerSettings.ShowEvents {
-		events, _ = searchEvents(d.client.CoreV1(), cronJobBeta, describerSettings.ChunkSize)
-	}
-	return describeCronJobBeta(cronJobBeta, events)
+	return "", err
```

Comment:
I think it would be much more idiomatic to inverse the `if` condition:

```go
if err != nil {
    return "", err
}

if describerSettings.ShowEvents {
    events, _ = searchEvents(d.client.CoreV1(), cronJob, describerSettings.ChunkSize)
}
return describeCronJob(cronJob, events)
```

Reply:
Done. Thanks for review.